keras/engine/topology.py

```diff
@@ -2704,7 +2704,7 @@ def to_yaml(self, **kwargs):
         """
         return yaml.dump(self._updated_config(), **kwargs)
 
-    def summary(self, line_length=None, positions=None, print_fn=print):
+    def summary(self, line_length=None, positions=None, print_fn=None):
```

Comment:
The docstring should state that `print_fn` defaults to `print` if unspecified.

Reply:
Updated, @fchollet.